lib/user.rb

```diff
@@ -0,0 +1,29 @@
+require 'pry'
+
+
+class User
+
+attr_reader :name, :jokes
+
+  def initialize(name)
+    @name = name
+    @jokes = []
+  end
+
+  def learn(joke)
+    @jokes << joke
+  end
+
+  def tell(who, what)
```

Comment:
I'd probably name this a little more specifically-- maybe even something like `user, joke`-- since that's technically what the "who" and "what" actually are.